lib/config/presets/github.ts

```diff
@@ -7,12 +7,40 @@ import { PLATFORM_TYPE_GITHUB } from '../../constants/platforms';
 
 const http = new Http(PLATFORM_TYPE_GITHUB);
 
-async function fetchJSONFile(
+export function setInternalPreset(content: { body: Preset }): void {
+  global.repoCache.internalPresets = content;
```

Comment:
```suggestion
  global.repoCache.internalPresets = Promise.resolve(content);
```

So we always have a promise